README.md

```diff
@@ -19,17 +19,48 @@ $> npm install react-onclickoutside --save
 var onClickOutside = require('react-onclickoutside');
 
 // create a new component, wrapped by this onclickoutside HOC:
-var MyComponent = onClickOutside(React.createClass({
+var MyComponent = onClickOutside(
```

Comment:
this gets unwieldy. Making the function take an optional object would be a better API, making it support both `onClickOutside(ReactComponent)` calls as well as `onClickOutside(ReactComponent, optionsObject)`